Files/Views/MainPage.xaml.cs

```diff
@@ -75,7 +73,7 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
         {
             if (eventArgs.NavigationMode != NavigationMode.Back)
             {
-                App.AppSettings = new SettingsViewModel();
+                App.AppSettings = await SettingsViewModel.CreateInstance();
```

Comment:
Mmm this adds 2 seconds of white screen before anything is loaded, not good.
If we remove drives from settings then loading drives should be done either at the end

Reply:
@gave92 does the latest commit give you the same white screen?
Creatings SettingsViewModel is expensive, and I am already testing some ideas to split DrivesManager and CloudDrivesManager out and make them properties of App; with all dependencies being constructed in parallel to minimise startup time.